src/App.vue

```diff
@@ -27,7 +28,7 @@ export default
         }
         else
         {
-            // setting to central London
+            // setting to central London or gues location by id
             Store.dispatch('setLocation', { latitude: 52, longitude: -0.12 });
```

Comment:
This can be set as default in the state, since it's basically default. Later on we can move it to action logic and guess user location from either last gathered location / IP/ sth elese